lib/logger.js

```diff
@@ -0,0 +1,3 @@
+var logger = require('@google/cloud-diagnostics-common').logger;
+var logLevel = process.env.GCLOUD_ERRORS_LOGLEVEL;
```

Comment:
@ofrobots @matthewloring The default behavior of the logger is not log given that it the log level is undefined:
https://github.com/GoogleCloudPlatform/cloud-diagnostics-common-nodejs/blob/master/lib/logger.js#L48

Therefore this behavior should match the description right?